guides/common/modules/proc_assigning-convert2rhel-role-to-host.adoc

```diff
@@ -9,3 +9,8 @@ Use this procedure to assign the Ansible Role to the host.
 . Click the *Ansible Roles* tab and from the *Available Ansible Roles* list, search for the `{ansible-namespace}.convert2rhel` role.
 . Click the `+` icon to add it to the *Assigned Ansible Roles* for the host.
 . Click *Submit*.
+
+[NOTE]
+====
+Do not run the Ansible role before you create required variables by following xref:creating_variables_for_ansible_role_{context}[].
```

Comment:
I am a bit hesitant to add this note here.

First, it could also be a warning because the Ansible role will definitly not work without the variables; second because I am unsure this is the correct place. Is it maybe more of a prerequisite? Or maybe, as alternative, add "Continue by adding the required variables. For more information, see xref..."?

Reply:
Understood. I reworded the message and added it as the next step in the procedure.